openapi/components/schemas/TagUntag/TagUntagRule.yaml

```
@@ -0,0 +1,94 @@
+type: object
+description: Tag / Untag rule.
+required:
+  - name
+  - eventType
+  - addingTags
+  - removingTags
+properties:
+  id:
+    description: Unique rule identifier string.
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    description: Rule's name.
```

issue: where possible, avoid [possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
    description: Name of the rule.
```